Backend/app/services/alert_monitoring_service.py

```
@@ -0,0 +1,220 @@
+            
+            # Get ROI metrics
+            try:
+                roi_data = self.roi_service.calculate_roi(campaign_id)
+                if roi_data:
+                    metrics.update({
+                        'roi': roi_data.get('roi', 0),
+                        'revenue': roi_data.get('revenue', 0),
+                        'cost_per_acquisition': roi_data.get('cost_per_acquisition', 0),
+                        'conversion_rate': roi_data.get('conversion_rate', 0)
+                    })
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)